node/impl/full/eth.go

```diff
@@ -1957,7 +1958,8 @@ func (e *EthEventHandler) EthNewFilter(ctx context.Context, filterSpec *ethtypes
 		return ethtypes.EthFilterID{}, err
 	}
 
-	f, err := e.EventFilterManager.Install(ctx, pf.minHeight, pf.maxHeight, pf.tipsetCid, pf.addresses, pf.keys, true)
```

Comment:
change in functionality here - you couldn't use `eth_newFilter` to install a filter that got you a reverted tipset but now you can; but I think this is reasonable and maybe we consider this a bug being fixed

Reply:
Filters also exist to observe real time changes to chain state and so reverts should be a part of it.